rita/src/rita_client/traffic_watcher/mod.rs

```diff
@@ -101,16 +92,52 @@ pub fn watch<T: Read + Write>(
     trace!("exit price {}", exit_price);
 
     if destinations.contains_key(&exit.mesh_ip) {
+        let client = reqwest::Client::builder()
+            .timeout(Duration::from_secs(5))
```

Comment:
maybe a shorter timeout is better, because we have this running every 5 seconds and we don't want them to pile up

Reply:
Also maybe async would be a good thing to add here, as we don't want to block too much, it's a bit tricky in rust but let me know if you want to give it a shot

Reply:
@kingoflolz Long RTTs will be entirely possible in congested networks. As for async the delay between taking the client_tx timestamp and the actual transfer of the request is important to be minimized - introducing an additional potentially non-deterministic delay between `client_tx` assignment and when `futures` schedules to make the request would be a big footshot.

Reply:
it might be better to have a thread and block, just to make sure we don't get into any futures bs.

Reply:
I'm using `reqwest` synchronously and this fragment of code doesn't depend on anything async, presently async problems shouldn't be the case here.

Reply:
makes sense